packages/astro/vendor/vite/dist/client/client.mjs

```diff
@@ -195,20 +195,15 @@ function warnFailedFetch(err, path) {
         `This could be due to syntax errors or importing non-existent ` +
         `modules. (see errors above)`);
 }
+function cleanUrl(pathname) {
```

Comment:
Note that this is the output of `pnpm run build` in the Vite repo. These files are generated from the build.